forge-gui/res/cardsfolder/e/endless_swarm.txt

```diff
@@ -6,4 +6,6 @@ A:SP$ Token | Cost$ 5 G G G | TokenAmount$ X | TokenScript$ g_1_1_snake | TokenO
 SVar:X:Count$InYourHand
 #NOTE: The AI will not stop making land drops after casting this spell (so, random decks may also utilize manlands and other cards with activation cost already on the battlefield to the maximum effect)
 AI:RemoveDeck:Random
+#NOTE: Memory leak.
```

Comment:
What exactly is meant here? What happens exactly?

Reply:
Tested it and it seems like it worked correctly for me. It does produce a lot of tokens so low-memory systems may run out of memory, yes, but it's not necessarily a problem with the AI handling of the card. Increasing the Java memory heap size may help on systems that have a lot of RAM. Not sure if maybe you meant something else though?

Reply:
![image](https://github.com/Card-Forge/forge/assets/10385459/8153a5c7-55fa-4570-a7b3-5133d2701c6b)

The only abilities the opponent had here were mana abilities, and it was slowing down every single step, not just the combat steps. That tells me the AI was *considering* what to do when it had *nothing* it could do.